prep-html-script-tag/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Script Tag</title>
+  </head>
+  <body>
+    <script>
+      console.log('I have added JS to a script tag in the HTML!');
+      console.log('I have added a script file via the src attribute!');
```

Comment:
This `console.log('I have added a script file via the src attribute!');` should be in *main.js*

Reply:
Just edited this, hope it's finally right!